scripts/mysqld_safe.sh

```diff
@@ -21,9 +21,11 @@ mysqld_ld_library_path=
 flush_caches=0
 numa_interleave=0
 wsrep_on=0
+simulate=0
```

Comment:
I'd gladly deprecate it. I understand systemd is good at running system server instances. But can it run instances per regular user demand?